notebooks/tsp.ipynb

```diff
@@ -1449,16 +1458,139 @@
    "execution_count": 16,
    "metadata": {
     "ExecuteTime": {
-     "end_time": "2021-06-28T23:35:26.687336Z",
-     "start_time": "2021-06-28T23:35:25.886048Z"
+     "end_time": "2022-04-16T21:09:15.957219Z",
+     "start_time": "2022-04-16T21:09:15.576710Z"
     }
    },
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "x_0,1: 1.0\n",
+      "Welcome to the CBC MILP Solver \n",
```

Comment:
This is a hefty output that wasn't here before. Is it okay to print all of this by default?

Reply:
Good point. Will push up a fresh version.